tests/set-labels/empty-resource/Kptfile

```diff
@@ -0,0 +1,10 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-labels:unstable
+      configPath: fn-config.yaml
+      selectors:
```

Comment:
You don't have to specify the selectors here, right?

Reply:
Yes, I am waiting for local-config PR to merge first, then I will make some changes regarding those tests